config/locales/en.yml

```diff
@@ -76,7 +76,7 @@ en:
     service_provider_deleted: Success! You have deleted %{issuer}
     service_provider_saved: Success! You have saved %{issuer}
     service_provider_iaa_notice:
-        All configuration changes or new configurations must be promoted to production following the process outlined in the <a href="https://developers.login.gov/production" target="_blank">Login.gov Developer Docs</a> - configurations must be reviewed by the Login.gov Partnerships team and approved by Friday in order to go live the following Thursday. We suggest allowing two weeks for integration launch or change requests.
+        All configuration changes or new configurations must be promoted to production following the process outlined in the <a href="https://developers.login.gov/production" target="_blank">Login.gov Developer Docs</a>. <b>Please allow two weeks for integration launch or change requests.</b>
```

Comment:
[question, non blocking] do we also want to include a note about how push notification urls take three weeks?